examples/person-detection/person_detection.py

```diff
@@ -23,12 +24,35 @@
 !mkdir -p binaries/person_detection && cd binaries/person_detection && wget https://github.com/antmicro/tensorflow-arduino-examples-binaries/raw/master/person_detection/person_detection.ino.elf # fetch prebuilt binaries
 
 # %% [markdown]
-"""## Take a photo"""
+"""
+## Take a photo
+To change option rerun the cell
+# """
 
 # %%
+import ipywidgets as w
 from IPython.display import Image
-photo = take_photo()
-display(Image(photo))
+
+def handle_selection(obj):
+  if obj['new'] == 'Camera':
+    photo = take_photo()
+    display(Image(photo))
+  elif obj['new'] == 'Upload':
+    display(uploadButton)
+    uploadButton.observe(handle_upload, 'data')
```

Comment:
Shouldn't you first configure the `uploadButton` before displaying it?

Reply:
It does work for me, but it's strange. I have to click upload, the buttons change, then I click upload again...

Reply:
It actually is configured first, we're in a function here

Reply:
Oh, I understand - you mean switching these two lines. Makes sense

Reply:
That's exactly what I meant. That there *might* be a situation when I see the button and interact with it before the `handle_upload()` callback is registered. Not sure how probable it is, but still.